packages/angular/src/ng-add-setup-project/files/src/app/coveo.components.module.ts

```diff
@@ -0,0 +1,49 @@
+import {NgModule} from '@angular/core';
```

Comment:
Why are we seeing changes in the angular packages, that's just moving around some import ?

Is this necessary/mandatory, or just because it's nicer this way. I'm not sure why changing from lerna would affect this ?

Because if it's the latter... please limit those to more restricted changes or dedicated PR, and not part of a PR that's already 100 file changes 😅 

However, with that said, it is much cleaner like this.

Reply:
I had issue extracting the changeset and threw the towel yesterday evening, but well, done #641